docs/docs/channels/sms/kavenegar.md

```diff
@@ -0,0 +1,24 @@
+# SMS77
+
+It is possible to use the [KaveNegar](https://www.kavenegar.com) provider to send SMS messages to the customers using the Novu Platform with a single API to create multi-channel experiences.
```

Comment:
I am sorry to be a pain but I didn't know that the product name is Kave Negar. If so the renaming should be `kaveNegar` when camelcasing and `kave-negar` when kebab-casing. I will try to leave you where needs to be changed. Apologies for not realising that. 🙏🏻

Reply:
Dear @p-fernandez
"kavenegar" is correct
"kave" ane "negar" are two different words in persian language, but, when we are writing about it, call it as "kavenegar". 
it s not require to rename "kavenegar" to "kave-negar"."kavenegar" is correct.